communication/communication-core/src/main/java/org/eclipse/jnosql/metamodel/DefaultAttribute.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 Otávio Santana and others
+ *   All rights reserved. This program and the accompanying materials
+ *   are made available under the terms of the Eclipse Public License v1.0
+ *   and Apache License v2.0 which accompanies this distribution.
+ *   The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ *   and the Apache License v2.0 is available at http://www.opensource.org/licenses/apache2.0.php.
+ *
+ *   You may elect to redistribute this code under either of these licenses.
+ *
+ *   Contributors:
+ *
+ *   Alessandro Moscatelli
+ */
+package org.eclipse.jnosql.metamodel;
+
+import jakarta.nosql.metamodel.Attribute;
+import org.eclipse.jnosql.AbstractGenericType;
+
```

Comment:
In general, when the class is public it would be good if it does provide a documentation, JavaDoc.

Reply:
done !